### PR TITLE
Reimbursement tutorial

### PR DESCRIPTION
**Pull request by danich1**
I think the onboarding documents should have a reimbursement tutorial. This process isn't easy, so this PR should fix that issue. Will need a non-graduate student to fill out the PostDoc/Staff section of the agreement form. Any problems or suggested changes are happily welcomed!

### CODE REVIEW

**Comment by cgreene** [comment]
Wow - this process is onerous for students. I think @arielah just did one of these. @arielah, when you get a chance can you look at the steps and see if this aligns with what you also had to do and/or if there are any other instructions that should be added? We'll get the student one merged first, and then do the faculty/staff/postdoc one.

**Comment by arielah** [comment]
Looks good to me, and it matches the process I (circuitously) followed. Thanks for putting this together, David!

File: extras/infrastructure.md [hunk 1]
```diff
@@ -42,6 +42,7 @@ Under **Profile Settings** > **Personal Information**, fill in required informat
 Set the default Travel Approver to **Jason Molli**.
```
Comment (cgreene) [inline extras/infrastructure.md:42]: Note to self - this may no longer be true.

File: extras/reimbursement_tutorial.md [hunk 2]
```diff
@@ -0,0 +1,130 @@
+# Filling Out A Reimbursement 101
+
+This is a tutorial on how to fill out a reimbursement. May it be for travel or for hosting a candidate interview dinner.
```
Comment (cgreene) [inline extras/reimbursement_tutorial.md]:
```suggestion
This is a tutorial on how to fill out a reimbursement for travel or for hosting a candidate interview dinner.
```

File: extras/reimbursement_tutorial.md [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+For this tutorial we will assume that we are reimbursing a postdoc candidate interview dinner.
+
+## Fill Out Agreement Form
+
+### Graduate Student
+
+First step is to fill out an [agreement form](https://www.finance.upenn.edu/sites/default/files/Penn-Student-Agreement.pdf).
```
Comment (cgreene) [inline extras/reimbursement_tutorial.md]:
```suggestion
The first step is to fill out an [agreement form](https://www.finance.upenn.edu/sites/default/files/Penn-Student-Agreement.pdf). 
```

File: extras/reimbursement_tutorial.md [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+
+This is a tutorial on how to fill out a reimbursement. May it be for travel or for hosting a candidate interview dinner.
+For this tutorial we will assume that we are reimbursing a postdoc candidate interview dinner.
+
+## Fill Out Agreement Form
+
+### Graduate Student
+
+First step is to fill out an [agreement form](https://www.finance.upenn.edu/sites/default/files/Penn-Student-Agreement.pdf). 
+This form is essential for getting any reimbursements accept.
```
Comment (cgreene) [inline extras/reimbursement_tutorial.md]:
```suggestion
This form is essential for getting any reimbursements accepted.
```

File: extras/reimbursement_tutorial.md [hunk 5]
```diff
@@ -0,0 +1,130 @@
...
+**Any submitted report that misses this form will bounce back**.
+Filling out this form is pretty straight forward.
+For the scope of work section fill out topic keywords that describe of your thesis project (PhD Candidates) or current research (Pre-Candidacy Students).
```
Comment (cgreene) [inline extras/reimbursement_tutorial.md]:
```suggestion
For the scope of work section, fill out topic keywords that describe of your thesis project (PhD Candidates) or current research (Pre-Candidacy Students).
```

File: extras/reimbursement_tutorial.md [hunk 6]
```diff
@@ -0,0 +1,130 @@
...
+Your screen should look like the screenshot below minus the submitted report.
+
+![opening screen](tutorial_screenshots/new_report_screen.png)
+
+Click on the create new report to get started on filling the reimbursement form out.
+Your screen should look like below.
+
+![report header](tutorial_screenshots/blank_report_header.png)
+
+Now the fields highlighted in red are what you need to fill out.
```
Comment (cgreene) [inline extras/reimbursement_tutorial.md]:
```suggestion
Fill out the fields highlighted in red. 
```

File: extras/reimbursement_tutorial.md [hunk 7]
```diff
@@ -0,0 +1,130 @@
...
+Your screen should look like below.
+
+![report header](tutorial_screenshots/blank_report_header.png)
+
+Now the fields highlighted in red are what you need to fill out. 
+**Type of Travel/Reimbursement**, **Report Name** and **Report Date** are straight forward to fill out.
+The policy field should be automatically filled, so don't worry about that.
+Now an important field to fill out is the **Business Justification**.
+In this field you need to provide justification for why you should be reimbursed.
+Rule of thumb: more detail you prove the better.
```
Comment (cgreene) [inline extras/reimbursement_tutorial.md]:
```suggestion
Rule of thumb: more detail you provide the better.
```

File: extras/reimbursement_tutorial.md [hunk 8]
```diff
@@ -0,0 +1,130 @@
...
+For example, if this is a travel reimbursement for a conference, you *should* provide the following information:
```
Comment (cgreene) [inline extras/reimbursement_tutorial.md]:
```suggestion
For example, if this is a travel reimbursement for a conference, you *must* provide the following information:
```
They made this entirely mandatory now. Mine get rejected without it too.

File: extras/reimbursement_tutorial.md [hunk 9]
```diff
@@ -0,0 +1,130 @@
...
+
+1. Name of Conference
+2. Conference Location
```
Comment (cgreene) [inline extras/reimbursement_tutorial.md]: Do you also have to put in the dates? We do.

File: extras/reimbursement_tutorial.md [hunk 10]
```diff
@@ -0,0 +1,130 @@
...
+3. Why you attended the conference - i.e. Present work via poster or talk 
+4. If PhD Candidate make sure to reference that presented work pertains to a specific aim of your thesis.
+5. For other students talk about how this is your current research project.
+6. For staff just provide as much detail as you can.
+
+After you fill out the Business Justification section, select the appropriate value for the School/Center section.
+For this lab select **(40) MED**.
+Here comes the tricky part: selecting the correct grant number.
+Please take a look at [greenelab funds](https://github.com/greenelab/funds) to find the appropriate grant number.
+If not sure what number is appropriate, ask a memeber of the lab.
```
Comment (cgreene) [inline extras/reimbursement_tutorial.md]:
```suggestion
If not sure what number is appropriate, ask Casey.
```

File: extras/reimbursement_tutorial.md [hunk 11]
```diff
@@ -0,0 +1,130 @@
...
+![main report](tutorial_screenshots/main_report_page.png)
+
+Congratulations!! You just fill out your expense report for dinner.
+Repeat the above process if you have more expenses that need to be reimbursed.
+Now last step is to attach that [agreement form](#fill-out-agreement-form) to this report.
+Look at this screenshot:
+
+![attach receipt](tutorial_screenshots/attach_receipts.png)
+
+After clicking the option, navigate to the scanned  agreement form and attached it.
```
Comment (cgreene) [inline extras/reimbursement_tutorial.md]:
```suggestion
After clicking the option, navigate to the scanned agreement form and attach it.
```

File: extras/reimbursement_tutorial.md [hunk 12]
```diff
@@ -0,0 +1,130 @@
...
+**Do not add it as an expense**.
+If you do, the report has a high likelihood of being bounced back.
+Once you have attached the form, hit the submit button (top right) and play the waiting game.
+Usually it can take a few days to a few weeks.
+All depends on how quickly the report gets approved.
```
Comment (cgreene) [inline extras/reimbursement_tutorial.md]:
```suggestion
The time depends on how quickly the report gets approved.
It is possible that you or Casey can speed the process up: please let him know if a reimbursement gets delayed.
```

**Review by cgreene — APPROVED** [review]
A few changes I'd like to see. Is Jason Molli still the person you put in as a travel approver? I don't think he works for the department anymore, but I may be wrong.

File: extras/reimbursement_tutorial.md (hunk 9 above)
Comment (danich1) [reply extras/reimbursement_tutorial.md]: Usually the date is in the report header, but I added it to the instructions just to be on the safe side.

**Pull request merged**